Doesn't build out of the box

Hi.

I just checked out the last commit. SHA: 5cfe348826b4c7d4570ab1982fb8643812a1ec07

It doesn't build out of the box.

```console
me@home ~/distroless (main)> bazel build //...
Starting local Bazel server and connecting to it...
INFO: Analyzed 2277 targets (549 packages loaded, 34144 targets configured).
INFO: Found 2277 targets...
ERROR: /home/me/distroless/base/BUILD:102:19: GoCompilePkg base/check_certs_ppc64le_debian11.a failed: (Exit 1): builder failed: error executing command (from target //base:check_certs_ppc64le_debian11) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -installsuffix linux_ppc64le -src base/testdata/check_certs.go -embedroot '' -embedroot ... (remaining 19 arguments skipped)
Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
base/testdata/check_certs.go:18:2: could not import crypto/x509 (open bazel-out/k8-fastbuild-ST-28dcfbc35f29/bin/external/io_bazel_rules_go/stdlib_/pkg/linux_ppc64le/crypto/x509.a: no such file or directory)
base/testdata/check_certs.go:19:2: could not import fmt (open bazel-out/k8-fastbuild-ST-28dcfbc35f29/bin/external/io_bazel_rules_go/stdlib_/pkg/linux_ppc64le/fmt.a: no such file or directory)
base/testdata/check_certs.go:20:2: could not import os (open bazel-out/k8-fastbuild-ST-28dcfbc35f29/bin/external/io_bazel_rules_go/stdlib_/pkg/linux_ppc64le/os.a: no such file or directory)
compilepkg: error running subcommand external/go_sdk/pkg/tool/linux_amd64/compile: exit status 2
ERROR: /home/me/.cache/bazel/_bazel_me/8ece7ddcfdfdb75817ed200d40df1593/external/io_bazel_rules_go/BUILD.bazel:42:7: Error while validating output TreeArtifact File:[[<execution_root>]bazel-out/k8-opt-exec-2B5CBBC6-ST-b33d65c724e6/bin]external/io_bazel_rules_go/stdlib_/pkg : java.lang.InterruptedException
ERROR: /home/me/.cache/bazel/_bazel_me/8ece7ddcfdfdb75817ed200d40df1593/external/io_bazel_rules_go/BUILD.bazel:42:7: Error while validating output TreeArtifact File:[[<execution_root>]bazel-out/k8-fastbuild-ST-8ee1a4b74a6f/bin]external/io_bazel_rules_go/stdlib_/pkg : java.lang.InterruptedException
ERROR: /home/me/.cache/bazel/_bazel_me/8ece7ddcfdfdb75817ed200d40df1593/external/io_bazel_rules_go/BUILD.bazel:42:7: Error while validating output TreeArtifact File:[[<execution_root>]bazel-out/k8-fastbuild-ST-84b95c4f6491/bin]external/io_bazel_rules_go/stdlib_/pkg : java.lang.InterruptedException
ERROR: /home/me/.cache/bazel/_bazel_me/8ece7ddcfdfdb75817ed200d40df1593/external/io_bazel_rules_go/BUILD.bazel:42:7: Error while validating output TreeArtifact File:[[<execution_root>]bazel-out/k8-fastbuild-ST-b33d65c724e6/bin]external/io_bazel_rules_go/stdlib_/pkg : java.lang.InterruptedException
ERROR: /home/me/.cache/bazel/_bazel_me/8ece7ddcfdfdb75817ed200d40df1593/external/io_bazel_rules_go/BUILD.bazel:42:7: Error while validating output TreeArtifact File:[[<execution_root>]bazel-out/k8-fastbuild/bin]external/io_bazel_rules_go/stdlib_/pkg : java.lang.InterruptedException
INFO: Elapsed time: 153.167s, Critical Path: 119.11s
INFO: 5729 processes: 5578 internal, 137 linux-sandbox, 14 worker.
FAILED: Build did NOT complete successfully
```

This is an ubuntu box. Ubuntu 22.04.3 LTS

I'm not proficient enough in bazel to decode that error further.

No release tags in the source tree ?

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -39,10 +39,10 @@ http_archive(
 # rules_go setup
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "278b7ff5a826f3dc10f04feaf0b70d48b68748ccd512d7f98bf442077f043fe3",
+    sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.41.0/rules_go-v0.41.0.zip",
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.41.0/rules_go-v0.41.0.zip",
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip",
     ],
 )
 
@@ -50,7 +50,7 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_depe
 
 go_rules_dependencies()
 
-go_register_toolchains(version = "1.21.0")
+go_register_toolchains(version = "1.19.1")
 
 http_archive(
     name = "bazel_gazelle",

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -37,6 +37,7 @@ http_archive(
 )
 
 # rules_go setup
+# NOTE: cannot upgrade past v0.35.0 until https://github.com/bazelbuild/rules_go/issues/3531 is resolved.
 http_archive(
     name = "io_bazel_rules_go",
     sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",

